test/installPSRP.sh

```diff
@@ -113,27 +117,27 @@ if [ "$isMacOS" = "true" ]; then
 	# Mac OS don't have mount_cifs, so use mount_smbfs
 	omifolder=$(get_omifolder "$omiversion" "$platfrom" "$opensslversion")
 	echo "mounting from $realdataDir folder to omi folder: $omifolder"
-	sudo mount -t smbfs '//redmond.corp.microsoft.com;scxsvc:Sep2016Q4!@wsscnafiler43/ostcdata$' /download
+	sudo mount -t smbfs '//redmond.corp.microsoft.com;scxsvc:'"$redmondpassword"'@wsscnafiler43/ostcdata$' /download
```

Comment:
Ugh, I missed this originally (Sep2016...). Please, never **EVER** check in password to GitHub. GitHub is "public", anybody can see it, and the corpnet isn't always as secure as we might like ... thanks.

Fixed now, but please keep this in mind in the future, thanks.